scripts/globals/mobskills/pollen.lua

```diff
@@ -24,17 +24,17 @@ mobskillObject.onMobWeaponSkill = function(target, mob, skill)
 
     if mob:getPool() == 385 then
         potency = 25
-        potency = potency - math.random(0,5)
+        potency = potency - math.random(0, 5)
     end
 
-    potency = potency - math.random(0, potency/4)
+    potency = potency - math.random(0, potency / 4)
 
     skill:setMsg(xi.msg.basic.SELF_HEAL)
 
     if mob:getPool() == 979 then -- Demonic Tiphia pollen recovers 3k+ HP
         return xi.mobskills.mobHealMove(mob, mob:getMaxHP() * potency / 30)
     else
-        return xi.mobskills.mobHealMove(mob, mob:getMaxHP() * potency / 100)
+        return xi.mobskills.mobHealMove(mob, mob:getMaxHP() * (147 / 1024))
```

Comment:
Potency isn't used for non-demonic tiphia and is static. is this intended?

Reply:
Looks like it from Jimmayus' notes but I don't understand this potency value anyway